keyserver/policy.go

```diff
@@ -1,14 +1,18 @@
 package keyserver
 
 import (
+	"io/ioutil"
 	"log"
 
 	"github.com/BurntSushi/toml"
 	"github.com/coniks-sys/coniks-go/merkletree"
+	"github.com/yahoo/coname/vrf"
 )
 
 type ServerPolicies struct {
 	EpochDeadline merkletree.TimeStamp `toml:"epoch_deadline"`
+	VRFKeyPath    string               `toml:"vrf_key_path"`
+	VRFKey        [vrf.SecretKeySize]byte
```

Comment:
I think the public key of the VRF key pair should be part of the ``Policies`` interface in ``merkletree/policy.go``